### PR TITLE
add(epp): oracle ucp jmx

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+---
+id: applications-oracle-ucp-jmx
+title: Oracle UCP JMX
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md:3]: @cg-tw do you think UCP is OK or should we have Universal Connection Pool?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+| *pool_name*#connection_pool.usage.count      |       |
+| *pool_name*#connection_pool.free.count       |       |
+| *pool_name*#connection_pool.usage.percentage | %     |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Veuiller installer l'agent Jolokia sur votre JVM [Jolokia download page](https://jolokia.org/download.html).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md:47]:
```suggestion
Veuillez installer l'agent Jolokia sur votre JVM: [Jolokia download page](https://jolokia.org/download.html).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+Demander à votre administrateur de le déployer et de vous fournir l'URL.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md:48]:
```suggestion
Demandez à votre administrateur de le déployer et de vous fournir l'URL.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+## Installation
+
+### Pack de supervision
+
+Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet
+n'est pas requise pour voir apparaître le pack dans le menu **Configuration > Plugin Packs > Gestionnaire**.
+
+Au contraire, si la plateforme utilise une licence *offline*, installez le paquet
+sur le **serveur central** via la commande correspondant au gestionnaire de paquet
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md:58]:
```suggestion
sur le **serveur central** via la commande correspondant au gestionnaire de paquets
```

File: pp/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+| *pool_name*#connection_pool.usage.count      |       |
+| *pool_name*#connection_pool.free.count       |       |
+| *pool_name*#connection_pool.usage.percentage | %     |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+Please install jolokia agent on your JVM [Jolokia download page](https://jolokia.org/download.html). Ask to your admin to deploy it and give you the URL.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md:47]:
```suggestion
Please install the Jolokia agent on your JVM: [Jolokia download page](https://jolokia.org/download.html). Ask to your admin to deploy it and give you the URL.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+title: Oracle UCP JMX
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Contenu du Pack
+
+### Modèles
+
+Le Pack Centreon **Oracle UCP JMX** apporte un modèle d'hôte :
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md:12]:
```suggestion
Le Pack Centreon **Oracle UCP JMX** apporte un modèle d'hôte pour la supervision d'**Oracle Universal Connection Pool** :
```

File: pp/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+title: Oracle UCP JMX
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack Assets
+
+### Templates
+
+The Centreon Pack **Oracle UCP JMX** brings a host template:
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-oracle-ucp-jmx.md:12]:
```suggestion
The Centreon Pack **Oracle UCP JMX** brings a host template to monitor **Oracle Universal Connection Pool**:
```

**Review by omercier — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**